app/src/app/constants/layers.ts

```diff
@@ -67,68 +134,73 @@ export function getBlocksLayerSpecification(
     layout: {
       visibility: 'visible',
     },
-    filter: getLayerFilter(layerId),
     paint: {
-      'line-opacity': ['case', ['boolean', ['feature-state', 'hover'], false], 1, 0.8],
-      'line-color': '#cecece',
+      'line-opacity': 1,
+      'line-color': [
+        'case',
+        ['boolean', ['feature-state', 'focused'], false],
+        '#000000', // Black color when focused
+        ['boolean', ['feature-state', 'highlighted'], false],
+        '#e5ff00', // yellow color when highlighted
+        '#000000', // Default color
+      ],
+      'line-width': [
+        'case',
+        ['boolean', ['feature-state', 'focused'], false],
+        10, // Width of 10 when focused
+        ['boolean', ['feature-state', 'highlighted'], false],
+        10, // Width of 5 when highlighted
+        0, // Default width
```

Comment:
PP: What do you think of using a round cap? Looks a little messy with the super thick line. Also have a personal pref to reduce the thickness a bit cause it's _really_ thick but won't die on that hill!

<img width="421" alt="image" src="https://github.com/user-attachments/assets/e154d0f8-474a-4664-84f4-dd427c0231be">

Reply:
Round cap is a good call!